modules/ref-dashboard-configuration-options.adoc

```diff
@@ -37,9 +37,17 @@ disableCustomServingRuntimes` | `false` | Shows the *Settings → Serving runtim
 | `dashboardConfig:
 disableDistributedWorkloads` | `false` | Shows the *Distributed workloads* option in the dashboard navigation menu. To hide this menu option, set the value to `true`.
 | `dashboardConfig:
+disableFineTuning` (Technology Preview) | `true` | Hides the the *Models → Model customization* option in the dashboard navigation menu, and the *LAB-tune* menu item and button for registered model versions. 
```

Comment:
should we specify the `LAB-tune` button? There are other buttons in registered model versions page.